enos/modules/vault_verify_raft_auto_join_voter/scripts/verify-raft-auto-join-voter.sh

```diff
@@ -47,4 +47,4 @@ export VAULT_ADDR='http://127.0.0.1:8200'
 
 # Retry a few times because it can take some time for things to settle after
 # all the nodes are unsealed
-retry 7 check_voter_status
+retry 10 check_voter_status
```

Comment:
Did you land on 10 from your experience?

Reply:
Yeah, I experienced a couple of times that 7 was not enough; 10 seems to be sufficient so far.